### ISSUE TITLE
ResourceValidator.ConstraintViolationImpl.errorMessage field may be final

### ISSUE THREAD

**Issue by manusa** [eclipse/jkube#2564]
### Description

The field [ResourceValidator.ConstraintViolationImpl.errorMessage](https://github.com/eclipse/jkube/blob/5da6068d7756784aef9569568ee80f98da97e296/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/validator/ResourceValidator.java#L213) may be 'final'.

The following lines
https://github.com/eclipse/jkube/blob/5da6068d7756784aef9569568ee80f98da97e296/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/validator/ResourceValidator.java#L213

- [x] should be changed to:
```java
        private final ValidationMessage errorMsg;
```

### Before you start :red_circle:

:point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down::point_down::point_down::point_down::point_down::point_down::point_down:
Make sure you read the [contributing guide](https://www.eclipse.org/jkube/contributing) first. Pay special attention to the ECA agreement section and the requirement to sign-off your commit.

#### Kubernetes

If you don't have a real Kubernetes cluster available (most probably), you can use Minikube or Kind to test with a local cluster.

- Minikube [get started guide](https://minikube.sigs.k8s.io/docs/start/)
- Kind (Requires Docker) [quick start](https://kind.sigs.k8s.io/docs/user/quick-start/)

#### OpenShift

If you don't have a real OpenShift cluster available (most probably), you can use [Red Hat's developer Sandbox for Red Hat OpenShift](https://developers.redhat.com/developer-sandbox). The only requirement is to have a Red Hat account.

Once you have your Sandbox environment, you'll need to download the oc tool from the cluster console.
(Press the `?` icon and from the context menu select `Command line tools`, you'll be redirected to https://$subdomain.openshiftapps.com/command-lines-tools where you'll be able to download the CLI for your platform)

**Comment by imlakshay08** [comment]
Hey @manusa , could assign this to me please. I wanna work on this one.

**Comment by imlakshay08** [comment]
@manusa , Thank you for assigning me. I have a small doubt though. It might seem silly, really sorry for that but, I am kinda confused. 
Few months I forked Jkube to make a contribution to it but due to some personal commitments, I kept on delaying it. But, to keep the track of latest repo, I used make to pull requests to my forked repo. Now, it shows "7 commits ahead" of this repo. 
So, my question is, should I delete my previously forked repo and fork a new one or I could just contribute in that old fork?
